tests/integration/targets/version_update/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+  vars:
+    cluster_update_config: "{{ sc_config[sc_host].cluster_update }}"
+
+  block:
+    - name: Fail test early if update is not allowed
+      ansible.builtin.assert:
+        that:
+          - cluster_update_config.magic_allow_string == "allow-cluster-update-test"
+    
+    - name: Get list of hc version updates
+      scale_computing.hypercore.version_update_info:
+      register: updates
+    
+    - name: Get initial version
+      scale_computing.hypercore.cluster_info:
+      register: cluster_initial
+    
```

Comment:
Here you can also look at version_update_status_info - check no update is in progress.

Reply:
added